packages/scripts/src/monitoring-dvm2.0/simulateVoting.ts

```diff
@@ -199,16 +198,15 @@ async function main() {
   const finalFee = (await store.computeFinalFee(votingToken.address)).rawValue;
 
   console.log(" 1. Funding foundation account...");
-  // Consolidate foundation balance with other large UMA holder account to be able to fund simulated voters.
-  // Make sure both accounts have enough ETH to pay for gas.
   const etherAmount = hre.ethers.utils.parseEther("10.0").toHexString();
-  await hre.network.provider.send("hardhat_setBalance", [accountWithUma, etherAmount]);
   await hre.network.provider.send("hardhat_setBalance", [foundationAddress, etherAmount]);
-  const accountWithUmaSigner = await hre.ethers.getImpersonatedSigner(accountWithUma);
   const foundationSigner = await hre.ethers.getImpersonatedSigner(foundationAddress);
-  await votingToken
-    .connect(accountWithUmaSigner)
-    .transfer(foundationAddress, await votingToken.balanceOf(accountWithUma));
+
```

Comment:
I've removed the `accountWithUma` thing because I wasn't able to simulate votes anymore due to the foundation wallet not reaching 50% of the staked tokens.